primer3/src/libprimer3/oligotm_main.c

```diff
@@ -166,7 +166,7 @@ main(int argc, char **argv)
           exit(-1);
         }
         i++;
-      } else if (!strncmp("-d", argv[i], 3)) {
+      } else if (!strncmp("-d", argv[i], 3)) { /* primer3-py bug-fix block need to parse 3 characters instead of 2 */
```

Comment:
In this case it should be 2 though, correct (`-d`)?

Reply:
Oh I see, to avoid matching `-dm` (realized this due to a similar but more verbose comment below, might be worth updating this comment to similarly point out the issue)

Reply:
My PR description points this out. I had previously changed this in past PR and this just adds a note to this.